index.js

```diff
@@ -48,6 +48,10 @@ var pluralTypes = {
     if (n % 10 === 1 && n % 100 !== 11) { return 0; }
     return n % 10 >= 2 && n % 10 <= 4 && (n % 100 < 10 || n % 100 >= 20) ? 1 : 2;
   },
+  lithuanian: function (n) {
+    if (n % 10 === 1 && n % 100 !== 11) { return 0; }
+    return n % 10 >= 2 && n % 10 <= 9 && (n % 100 < 10 || n % 100 >= 20) ? 1 : 2;
```

Comment:
maybe changing this from `< 10` and `>= 20` to `< 11` and `> 19` would be clearer?